woocommerce/woocommerce-gateway-stripe
# Update min supported version numbers

Also updates the PHPCS sniffer rules.


## Changes proposed in this Pull Request:

- Update minimum supported PHP version to version 7.0.
    - This is a sort of transitive dependency because we require at least WC 5.6 which requires at least PHP 7.0.
- Update the PHPCS php version sniff to version 7.0.
- Update minimum supported WP version in readme.txt to 5.6 instead of 4.6.
- Update some out of date version constants.

## Testing instructions

1. Try using the null-coalescing operator in a code editor.
    - For example; `$foo = 'bar' ?? 'baz';`
2. Make sure phpcs does not mark that as an issue.

---
